app/subscriber/src/components/tool-bar/AddToReportMenu.tsx

```diff
@@ -27,12 +27,17 @@ export const AddToReportMenu: React.FC<IAddToReportMenuProps> = ({ content }) =>
 
   /** Adds the content to the active report. */
   const addContentToReport = React.useCallback(
-    (section: string) => {
+    (sectionName: string) => {
       if (!activeReport) {
         return;
       }
 
-      const convertedContent = toInstanceContent(content, activeReport, section);
+      const convertedContent = toInstanceContent(
+        content,
+        activeReport.instances[0].id,
+        sectionName,
+        0,
```

Comment:
@Fosol, updated again. But quick question, how do we determine the sort order from here? Should it always go to the top/bottom?

Reply:
Does it make sense to have it as an argument here when it will apply to `1...n` pieces of content being passed?

Reply:
Sort order would default to the top (`sortOrder = 0`), but Scott has mentioned he wants a way to control this.  Perhaps focus on getting this to work before we provide a way for the user to choose the sort order value.